src/parser-babylon.js

```diff
@@ -2,7 +2,8 @@
 
 const createError = require("./parser-create-error");
 
-function parse(text) {
+function parse(text, options) {
```

Comment:
Hmm, after making `parseJavascriptExpression` use `babylon.parseExpression`, this is trickier than I thought, due to the way `multiparser` passes around options. For example, here's a patch that doesn't pass the tests:

<details>

```diff
diff --git a/src/multiparser.js b/src/multiparser.js
index b50c31b7..7b8602b8 100644
--- a/src/multiparser.js
+++ b/src/multiparser.js
@@ -229,7 +229,7 @@ function parseJavaScriptExpression(text, parsers) {
   return {
     type: "File",
     // Force parsing as an expression
-    program: parsers.babylon(text, { parseExpression: true })
+    program: parsers.babylon(text, true)
   };
 }
 
diff --git a/src/parser-babylon.js b/src/parser-babylon.js
index 2a004254..7f5d0766 100644
--- a/src/parser-babylon.js
+++ b/src/parser-babylon.js
@@ -2,8 +2,7 @@
 
 const createError = require("./parser-create-error");
 
-function parse(text, options) {
-  options = options || {};
+function parse(text, parseExpression) {
   // Inline the require to avoid loading all the JS if we don't use it
   const babylon = require("babylon");
 
@@ -26,7 +25,8 @@ function parse(text, options) {
     ]
   };
 
-  const parseMethod = options.parseExpression ? "parseExpression" : "parse";
+  console.log({ parseExpression });
+  const parseMethod = parseExpression ? "parseExpression" : "parse";
 
   let ast;
   try {
diff --git a/src/parser-json.js b/src/parser-json.js
index f7690ae1..ca9babe9 100644
--- a/src/parser-json.js
+++ b/src/parser-json.js
@@ -58,9 +58,10 @@ function toBabylon(node) {
         elements: node.children.map(toBabylon)
       });
     case "literal": {
-      const babylonExpression = require("./parser-babylon")(node.rawValue, {
-        parseExpression: true
-      });
+      const babylonExpression = require("./parser-babylon")(
+        node.rawValue,
+        true
+      );
       return Object.assign(babylonExpression, result, {
         type: babylonExpression.type
       });
```
</details>

Maybe @azz can help sort it out?